crates/compression/src/decompress.rs

```diff
@@ -211,6 +228,8 @@ where
         ctx: &DecompressCtx<D>,
     ) -> anyhow::Result<Self> {
         Ok(Transaction::mint(
+            // we should probably include the mint TxPointer in the compression if we decide to support
+            // multiple assets for mints, i.e more than 1 mint tx per block
```

Comment:
To support multiple assets to pay for fee, we will just introduce MintV2, with vector of assets and fee.

Reply:
addressed in [a379fdc](https://github.com/FuelLabs/fuel-core/pull/2593/commits/a379fdc24bea42b72ca5563db78636c086b22da0)